Sources/Afterpay/Model/PriceBreakdown.swift

```diff
@@ -35,7 +38,7 @@ struct PriceBreakdown {
 
     if let formattedPayment = formattedPayment, inRange {
       badgePlacement = .end
-      string = String(format: Strings.fourPaymentsFormat, formattedPayment)
+      string = String(format: Strings.fourPaymentsFormat, introText.rawValue, formattedPayment).trimmingCharacters(in: .whitespaces)
```

Comment:
Only thing I'd ask now is for you to please put a new line in to quash the swiftlint warning, please:

```suggestion
      string = String(format: Strings.fourPaymentsFormat, introText.rawValue, formattedPayment)
          .trimmingCharacters(in: .whitespaces)
```